src/OpenTelemetry.Instrumentation.Runtime/description.md

```diff
@@ -0,0 +1,198 @@
+| process.runtime.dotnet.**monitor.lock.contention.count**    | Monitor Lock Contention Count        | `{times}`   | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**threadpool.thread.count**          | ThreadPool Thread Count              | `{threads}` | ObservableGauge   | `Int32`    |                  |                  |
+| process.runtime.dotnet.**threadpool.completed.items.count** | ThreadPool Completed Work Item Count | `{items}`   | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**threadpool.queue.length**          | ThreadPool Queue Length              | `{items}`   | ObservableGauge   | `Int64`    |                  |                  |
+| process.runtime.dotnet.**active.timer.count**               | Number of Active Timers              | `{timers}`  | ObservableGauge   | `Int64`    |                  |                  |
+
+- [Monitor.LockContentionCount](https://docs.microsoft.com/dotnet/api/system.threading.monitor.lockcontentioncount?view=netcore-3.1):
+  Gets the number of times there was contention when trying to take the monitor's
+  lock.
+- [ThreadPool.ThreadCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.threadcount?view=netcore-3.1):
+  Gets the number of thread pool threads that currently exist.
+- [ThreadPool.CompletedWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.completedworkitemcount?view=netcore-3.1):
+  Gets the number of work items that have been processed so far.
+- [ThreadPool.PendingWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.pendingworkitemcount?view=netcore-3.1):
+  Gets the number of work items that are currently queued to be processed.
+- [Timer.ActiveCount](https://docs.microsoft.com/dotnet/api/system.threading.timer.activecount?view=netcore-3.1):
+  Gets the number of timers that are currently active. An active timer is registered
+  to tick at some point in the future, and has not yet been canceled.
+
+## Process related metrics
+
+The metrics in this section can be enabled by setting the
+`RuntimeMetricsOptions.IsProcessEnabled` switch.
+
+| Name                    | Description                            | Units          | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------|----------------------------------------|----------------|-------------------|------------|------------------|------------------|
+| process.cpu.utilization | CPU utilization of this process        | `1`            | ObservableGauge   | `Double`   |                  |                  |
+| process.cpu.time        | Processor time of this process         | `s`            | ObservableCounter | `Int64`    | state            | user, system     |
+| process.cpu.count       | The number of available logical CPUs   | `{processors}` | ObservableGauge   | `Int64`    |                  |                  |
```

Comment:
`process.cpu.count` metric was never officially added (see https://github.com/open-telemetry/opentelemetry-specification/pull/2392). I'll remove it later.

Reply:
Removed.